docs/scos/user/features/202200.0/promotions-discounts-feature-overview.md

```diff
@@ -141,18 +141,18 @@ The product selection based on the query string is dynamic:
 
 ### Promotional product
 
-Promotional product is a discount application type that discounts a particular product at a set quantity, enabling "buy X, get Y" promotions.
+Promotional product is a discount application type that discounts particular products at a set quantity, enabling "buy X, get Y" promotions.
 
-When a customer meets the conditions for a promotional product discount, the _Promotional products_ section is displayed in the cart and lets customers add the available quantity of the discounted product. The section consists of the product name, SKU, original and discounted price, and discount description.
+When a customer meets the conditions for a promotional product discount, the **Promotional products** section is displayed in the cart and lets customers add the available quantity of the discounted products. The section consists of the product name, SKU, original and discounted price, and discount description.
 
-![Promotional product](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/features/promotions-discounts-feature-overview.md/promotional-product-storefront.png)
+![Promotional product](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/features/promotions-discounts-feature-overview.md/202200.0/promotional-product-storefront.png)
 <!--
 old image: (https://spryker.s3.eu-central-1.amazonaws.com/docs/Features/Promotions+&+Discounts/Discount/Discount+Feature+Overview/promotional-product-frontend.png)
 -->
 
-A Back Office user selects the promotional product by entering an abstract product SKU in the Back Office. Also, they define the maximum quantity of the product to be sold with a discount.
+A Back Office user selects promotional products by entering comma-separated abstract product SKUs in the Back Office. Also, they define the maximum quantity of the products to be sold with a discount.
```

Comment:
Please find a suitable place in the doc and explain that the discount applies to one of the products, not all of them.

Reply:
Actually, the discount could apply to 1, 2, 10 products—the quantity depends on the dicount configiration.

Reply:
I think we could use an example here. For example, if a discount applies to 10 promotional products with quantity 3, a customer can select 3 units of any of the promotional products in any combination.